src/current/v24.3/ldap-authentication.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Configure LDAP Authentication
+summary: Learn how to configure CockroachDB to use LDAP for user authentication with directory services like Active Directory.
+toc: true
+---
+
+CockroachDB supports authentication and authorization using LDAP (Lightweight Directory Access Protocol) with services such as Active Directory. This allows you to integrate your existing directory service with CockroachDB for centralized user management and access control.
+
+This page describes how to configure CockroachDB user authentication using LDAP. You can also configure user authorization (role-based access control) with LDAP, to set users' CockroachDB roles based on group memberships in the directory, but this is not required.
```

Comment:
I got lost in the end of this para, but I think maybe it means that you can authenticate to your cluster using identities stored in LDAP, and you can assign them roles based on their group membership.  I wonder if you could assign them roles based on other criteria, such as their existence in the directory.